src/lib.rs

```diff
@@ -815,36 +813,34 @@ impl InflateStream {
-                                // 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, // 256-271
-                                // 7, 7, 7, 7, 7, 7, 7, 7, // 272-279
-                                // 8, 8, 8, 8, 8, 8, 8, 8, // 280-287
-                                // ]);
-                                // let dist = DynHuffman16::new(&[
-                                // 5, 5, 5, 5, 5, 5, 5, 5,
-                                // 5, 5, 5, 5, 5, 5, 5, 5,
-                                // 5, 5, 5, 5, 5, 5, 5, 5,
-                                // 5, 5, 5, 5, 5, 5, 5, 5
-                                // ]);
-                                // ok!(BlockDyn(lit, dist, 0))
-                                //
-                                ok!(BlockFixed)
+                                let lit = try!(DynHuffman16::new(&[
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 0-15
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 16-31
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 32-47
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 48-63
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 64-79
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 80-95
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 96-101
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 112-127
+                                    8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, // 128-143
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 144-159
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 160-175
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 176-191
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 192-207
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 208-223
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 224-239
+                                    9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, // 240-255
+                                    7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, // 256-271
+                                    7, 7, 7, 7, 7, 7, 7, 7, // 272-279
+                                    8, 8, 8, 8, 8, 8, 8, 8, // 280-287
+                                ]));
+                                let dist = try!(DynHuffman16::new(&[
+                                    5, 5, 5, 5, 5, 5, 5, 5,
+                                    5, 5, 5, 5, 5, 5, 5, 5,
+                                    5, 5, 5, 5, 5, 5, 5, 5,
+                                    5, 5, 5, 5, 5, 5, 5, 5
+                                ]));
```

Comment:
Actually, you should `.unwrap()`, sorry - these can't ever fail, they're effectively constants.

Reply:
But why it returns 'Result' then?

Reply:
I meant these calls can't ever fail because they have constant arguments - for other inputs, those functions may return `Err`.

Reply:
@eddyb done.